content/numpy/concepts/math-methods/terms/round/round.md

```diff
@@ -0,0 +1,155 @@
+---
+Title: '.round()' 
+Description: 'Rounds a number or an array of numbers to a specified number of decimal places.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'NumPy'
+  - 'Math'
+  - 'Arrays'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
+---
+
+**`.round()`** is a function from the NumPy Python library. It rounds a number or an array of numbers to a specified number of decimal places.
+
+## Syntax
+
+```psuedo
+numpy.round(array/number, decimal place, out)
+```
+
+- `array/number` is the number or array, whose elements (either a float or integer) will be rounded.
+- `decimal place` is the number of decimal places the number or elements in the array will be rounded to. The default value is 0. This parameter is optional and will accept negative integers as well as positive ones.
+- `out` is an optional output array which the result will feed into.
+
+> **Note:** Numpy is commonly imported as `import numpy as np`. This allows us to refer to `numpy` as `np`.
+> 
+> **Note:** `numpy.round()` will return an array without commas separating the elements. To see the output with commas use [`.repr()`](https://github.com/Codecademy/docs/blob/main/content/python/concepts/built-in-functions/terms/repr/repr.md).
+>
+> **Note:** `numpy.round()` will round to the nearest even number when exactly halfway between two numbers. 
+
+## Example 1
```

Comment:
We can have all the example use cases under one ##Example block only, which will be accompanied by one ```shell ``` block. Use comments to explain the codes (use cases)